dts/bindings/usb/nxp,kinetis-usbd.yaml

```diff
@@ -22,3 +22,9 @@ properties:
 
     interrupts:
       category: required
+
+    clocks:
```

Comment:
@pizi-nordic Should it be part of this commit?

Reply:
Yes, this commit also adds a couple of .yaml bindings

Reply:
No it just two, `nxp,kinetis-usbd.yaml` and `arm,cmsdk-watchdog.yaml`, the second is okay (v2m_musca-common.dtsi), but where is the `clocks` from `nxp,kinetis-usbd.yaml` used?

Reply:
Here: https://github.com/zephyrproject-rtos/zephyr/pull/16958/files#diff-be3b6b842e8552442c99fc8abcafeafd

Reply:
Yes, we had to split up the .dts and .yaml additions and the code changes to simplify the PR

Reply:
okay I understood the changes here.

Reply:
This is wrong, and needs a fix elsewhere